src/librustc_typeck/lib.rs

```diff
@@ -130,20 +130,20 @@ pub struct TypeAndSubsts<'tcx> {
     ty: Ty<'tcx>,
 }
 
-fn check_type_alias_enum_variants_enabled<'a, 'gcx, 'tcx>(tcx: TyCtxt<'a, 'gcx, 'tcx>,
-                                                          span: Span) {
-    if !tcx.features().type_alias_enum_variants {
-        let mut err = tcx.sess.struct_span_err(
-            span,
-            "enum variants on type aliases are experimental"
-        );
-        if nightly_options::is_nightly_build() {
-            help!(&mut err,
-                "add `#![feature(type_alias_enum_variants)]` to the \
-                crate attributes to enable");
-        }
-        err.emit();
-    }
+fn check_type_alias_enum_variants_enabled<'a, 'gcx, 'tcx>(_tcx: TyCtxt<'a, 'gcx, 'tcx>,
+                                                          _span: Span) {
+    // if !tcx.features().type_alias_enum_variants {
+    //     let mut err = tcx.sess.struct_span_err(
+    //         span,
+    //         "enum variants on type aliases are experimental"
+    //     );
+    //     if nightly_options::is_nightly_build() {
+    //         help!(&mut err,
+    //             "add `#![feature(type_alias_enum_variants)]` to the \
+    //             crate attributes to enable");
+    //     }
+    //     err.emit();
+    // }
```

Comment:
I presume you will uncomment this after you have run all the tests/checks you need to?

Reply:
Yes, this PR is necessary minimum to run crater, I'll do everything else after looking at the results.

Reply:
Sounds good.